website/docs/reference/widgets/table/column-settings.md

```diff
@@ -152,6 +153,24 @@ Based on the Column type, you can edit the cell content. Once done, you can move
 
 <VideoEmbed host="youtube" videoId="Wmitzz4UAGo" title="Editable" caption="Editable"/>
 
+### Column Freeze
+
+Column freeze controls whether the column can be frozen to the left, right or no freeze in the table. Once the column is frozen a Pin icon appears on the respective column in the Columns property.
```

Comment:
> Once the column is frozen a Pin icon appears on the respective column in the Columns property.
I think this can be updated to say, "Frozen columns will be pinned in the property pane"